entanglement_forging/core/wrappers/entanglement_forged_driver.py

```diff
@@ -48,26 +64,66 @@ def __init__(
-        q_molecule.eri = self._eri
-        q_molecule.num_molecular_orbitals = self._mo_coeff.shape[0]
-        q_molecule.num_alpha = self._num_alpha
-        q_molecule.num_beta = self._num_beta
-        q_molecule.nuclear_repulsion_energy = self._nuclear_repulsion_energy
+        # Create ParticleNumber property
+        particle_number = ParticleNumber(
+            self._mo_coeff.shape[0], (self._num_alpha, self._num_beta)
+        )
+
+        # Convert one body integrals from AO to MO basis
+        one_body_in_mo_basis = np.dot(
+            np.dot(np.transpose(self._mo_coeff), self._hcore), self._mo_coeff
+        )
+
+        # Convert two-body integrals from AO to MO basis
+        dim = self._eri.shape[0]
+        two_body_in_mo_basis = np.zeros((dim, dim, dim, dim))
+        for a_i in range(dim):
+            temp1 = np.einsum("i,i...->...", self._mo_coeff[:, a_i], self._eri)
+            for b_i in range(dim):
+                temp2 = np.einsum("j,j...->...", self._mo_coeff[:, b_i], temp1)
+                temp3 = np.einsum("kc,k...->...c", self._mo_coeff, temp2)
+                two_body_in_mo_basis[a_i, b_i, :, :] = np.einsum(
+                    "ld,l...c->...cd", self._mo_coeff, temp3
+                )
+
+        # Create ElectronicEnergy property with integrals in both bases and the nuclear repulsion energy
+        integrals: List[ElectronicIntegrals] = []
+        integrals.append(
+            OneBodyElectronicIntegrals(ElectronicBasis.AO, (self._hcore, None))
+        )
+        integrals.append(
+            TwoBodyElectronicIntegrals(
+                ElectronicBasis.AO, (self._eri, None, None, None)
+            )
+        )
+        integrals.append(
+            OneBodyElectronicIntegrals(ElectronicBasis.MO, (one_body_in_mo_basis, None))
+        )
+        integrals.append(
+            TwoBodyElectronicIntegrals(
+                ElectronicBasis.MO,
+                (
+                    two_body_in_mo_basis,
+                    two_body_in_mo_basis,
+                    two_body_in_mo_basis,
+                    None,
+                ),
+            )
+        )
+        electronic_energy = ElectronicEnergy(
+            integrals, nuclear_repulsion_energy=self._nuclear_repulsion_energy
+        )
 
-        one_body_in_mo_basis = QMolecule.oneeints2mo(self._hcore, self._mo_coeff)
-        q_molecule.mo_onee_ints = one_body_in_mo_basis
-        q_molecule.mo_onee_ints_b = one_body_in_mo_basis
+        # Define the transform from AO to MO
+        ele_basis_xform = ElectronicBasisTransform(
+            ElectronicBasis.AO, ElectronicBasis.MO, self._mo_coeff
+        )
 
-        two_body_in_mo_basis = QMolecule.twoeints2mo(self._eri, self._mo_coeff)
-        q_molecule.mo_eri_ints = two_body_in_mo_basis
-        q_molecule.mo_eri_ints_ba = two_body_in_mo_basis
-        q_molecule.mo_eri_ints_bb = two_body_in_mo_basis
+        result = ElectronicStructureDriverResult()
```

Comment:
I see that you are discarding the `Molecule` information. Is this because you only supplied them to have a working `QMolecule` in the past?
If not, you can follow the procedure here: https://github.com/Qiskit/qiskit-nature/blob/aae9c01b4d02bf5dbaa342c757be2cfef60a31d5/qiskit_nature/drivers/second_quantization/pyscfd/pyscfdriver.py#L528-L539

Note, that the `Molecule` is not a `Property` by itself and thus is set as an attribute of the `result` object.

Reply:
It looked like the atom_xyz and atom_symbol were left as empty lists, so I wasn't sure if there was ever any molecule geometry loaded into these objects. Am I missing something?

Reply:
I'm not sure tbh. If you don't really make use of them in this repo, they may well be empty and in that case there is no need for the `Molecule` object to be added to the `ElectronicStructureDriverResult` :+1: